exchange/events_test.go

```diff
@@ -71,6 +80,7 @@ func Test_eventsData_modifyBidJSON(t *testing.T) {
 		enabledForAccount bool
 		enabledForRequest bool
 		bidType           openrtb_ext.BidType
+		generateBidId     string
```

Comment:
Can you please rename this to `generatedBidID` as `generateBidId` indicates the `generateBidId` boolean flag rather than the generated BidID value?